integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/gcp/action/GcpClientActions.java

```diff
@@ -180,12 +183,17 @@ public void stopHostGroupInstances(List<String> instanceIds) {
         return operation == null ? null : operation.getError();
     }
 
-    public void listBucketSelectedObject(String baseLocation, String selectedObject, boolean zeroContent) {
+    public void listBucketSelectedObject(String baseLocation, String selectedObject, Boolean zeroContent) {
```

Comment:
`listBucketSelectedObject` has three implementations in IT module based on the provider:
- [S3ClientActions.listBucketSelectedObject(String baseLocation, String selectedObject, Boolean zeroContent)](https://github.com/hortonworks/cloudbreak/blob/bea15989757a942fe829eb61a4336cfd61de40a6/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/aws/amazons3/action/S3ClientActions.java#L72)
- [GcpClientActions.listBucketSelectedObject(String baseLocation, String selectedObject, boolean zeroContent)](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/gcp/action/GcpClientActions.java#L183)
- [AzureCloudBlobClientActions.listSelectedDirectory(String baseLocation, String selectedDirectory, Boolean zeroContent)](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/azure/azurecloudblob/action/AzureCloudBlobClientActions.java#L317)
In case of AWS and Azure the type was `Boolean` expect GCP where this was `boolean`. So I had to made a common solution in the [CloudFunctionality interface cloudStorageListContainer](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/CloudFunctionality.java#L53). Beyond this the `zeroContent` variable is required here. So cannot be `null`.

Reply:
If zeroContent cant be null, then we can use  boolean instead of Boolean. There is a potential NPE right now if somebody calls the method with null value.